addons/sale/tests/test_sale_order_down_payment.py

```diff
@@ -0,0 +1,228 @@
+from odoo.tests import tagged
+
+from .common import TestSaleCommon
+
+# pylint: disable=C0326
```

Comment:
?

Reply:
This disable pylint check on spaces. It prevents the column formatting of my expected values to trigger ci-style check on runbot.

Reply:
Use another way to indent/display/implement your tests but do not use pylint overrides, those are reserved for cases where it's really necessary (and targeted on specific lines).

Reply:
> Use another way to indent/display/implement your tests but do not use pylint overrides, those are reserved for cases where it's really necessary (and targeted on specific lines).

For what i can see, it's not uncommon in odoo tests (but declared just before the concerning lines, i can do that if you prefer)

Reply:
Well, only in accounting tests :D.
Yes, I'd prefer if it's only specified in limited places where it's really useful :)